src/Parser/RestoDruid/Modules/Features/Mastery.js

```diff
@@ -0,0 +1,143 @@
+import {formatPercentage} from 'common/format';
+import Module from 'Parser/Core/Module';
+import SPELLS from 'common/SPELLS';
+import Combatants from 'Parser/Core/Modules/Combatants';
+
+import {ABILITIES_AFFECTED_BY_HEALING_INCREASES} from '../../Constants';
+import {HEALS_MASTERY_STACK} from '../../Constants';
+
+const MASTERY_BONUS_FROM_ONE_RATING = 1 / 66666.6666666;
+const BASE_MASTERY_PERCENT = 0.048;
+
+class Mastery extends Module {
+  static dependencies = {
+    combatants: Combatants,
+  };
+
+  totalNoMasteryHealing = 0;
+  druidSpellNoMasteryHealing = 0;
+  masteryTimesHealing = 0;
+
+  hotHealingMap = null;
+  masteryBuffs = null;
+
+  on_initialized() {
+    // TODO use JS objects instead of Maps for hotHealingMap and masteryBuffs?
+    this.hotHealingMap = new Map();
+    for(const healId of HEALS_MASTERY_STACK) {
+      this.hotHealingMap.set(healId, {'name':SPELLS[healId].name, 'direct':0, 'mastery':0});
+    }
+
+    this.masteryBuffs = new Map([
+        [ SPELLS.ASTRAL_HARMONY.id, { 'spell':SPELLS.ASTRAL_HARMONY, 'amount':4000 } ],
```

Comment:
Setting the `spell` property here seems unnecessary as the key already allows you to fetch it (since SPELLS is also indexed by `id` so that you can access spells by id during runtime).

Reply:
Was over-designed, will fix with next PR